PayPalMessaging/src/main/java/com/braintreepayments/api/PayPalMessagingAnalytics.kt

```diff
@@ -0,0 +1,7 @@
+package com.braintreepayments.api;
+
+internal enum class PayPalMessagingAnalytics(val value: String) {
+    STARTED("paypal-messaging:create-view:started"),
+    FAILED("paypal-messaging:create-view:failed"),
+    SUCCEEDED("paypal-messaging:create-view:succeeded")
+}
```

Comment:
```suggestion
internal object PayPalMessagingAnalytics {
    const val STARTED = "paypal-messaging:create-view:started"
    const val FAILED = "paypal-messaging:create-view:failed"
    const val SUCCEEDED = "paypal-messaging:create-view:succeeded"
}
```
🧶 : I forgot in the past we've moved to this style (Tim's [PR reminded me](https://github.com/braintree/braintree_android/pull/908/files)). It's helpful because it's a plain String constant, so it allows us to omit `.value` when we call `sendAnalyticsEvent()`.